src/electionguard-ui/ElectionGuard.UI/ViewModels/LoginViewModel.cs

```diff
@@ -6,12 +6,27 @@ public partial class LoginViewModel : BaseViewModel
 {
     public LoginViewModel(IServiceProvider serviceProvider) : base("UserLogin", serviceProvider)
     {
+        _timer.Tick += HandleDbPing;
+        _timer.Start();
+    }
+
+    ~LoginViewModel()
+    {
```

Comment:
is this function even needed?  might consider putting the code that is here and in the LeavingPage to a single function.

Reply:
I'm not certain how classes in MAUI get marked for GC, and this would be run by the finalizer queue to ensure there's not a dangling timer. Good call on extracting this